src/main/java/io/antmedia/statistic/StatsCollector.java

```diff
@@ -467,6 +477,29 @@ public static JsonObject getSysteMemoryInfoJSObject() {
 		jsonObject.addProperty(IN_USE_SWAP_SPACE, SystemUtils.osInUseSwapSpace());
 		return jsonObject;
 	}
+	
+	public static JsonObject getNativeMemoryInfoJSObject() {
+		JsonObject jsonObject = new JsonObject();
+		
+		long totalPhysicalBytes = Pointer.totalPhysicalBytes();
+		long availablePhysicalBytes = Pointer.availablePhysicalBytes();
+		
+		jsonObject.addProperty(FREE_NATIVE_MEMORY, availablePhysicalBytes);
+		jsonObject.addProperty(IN_USE_NATIVE_MEMORY, totalPhysicalBytes - availablePhysicalBytes);
```

Comment:
I don't think we need to send IN_USE_NATIVE_MEMORY.

Reply:
Yes, we don't need it. But it needs to should be there, due to custom developers.